components/layout/construct.rs

```diff
@@ -1316,6 +1316,39 @@ impl<'a, ConcreteThreadSafeLayoutNode: ThreadSafeLayoutNode>
         self.build_flow_for_block_like(flow, node)
     }
 
+    /// Builds a fragment for a node with 'display: inline-flex'.
+    // Shared code with build_fragment_for_inline_block
+    fn build_fragment_for_inline_flex(&mut self, node: &ConcreteThreadSafeLayoutNode)
```

Comment:
This function should probably reuse `build_fragment_for_inline_block`, but I'm not sure if we want to have a `build_fragment_for_inline_block_or_inline_flex` and pass a flag into it, or should we have two functions and extract the shared part into a helper function.